src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -480,9 +518,15 @@ class CollabWrapper extends PureComponent<Props, CollabState> {
 
   /** Getter of context value. Returned object is stable. */
   getContextValue = (): CollabAPI => {
-    this.contextValue = this.contextValue || ({} as CollabAPI);
+    if (!this.contextValue) {
+      this.contextValue = {} as CollabAPI;
+      Object.defineProperties(this.contextValue, {
+        isCollaborating: {
+          get: () => this.isCollaborating,
+        },
+      });
+    }
```

Comment:
How hard would it be to make it a function instead? I really don't like getters as they give the wrong mental model to people reading the code.

Reply:
good point, done!